package.json

```diff
@@ -15,6 +15,7 @@
     "github-release": "node scripts/github-release",
     "build-docs": "PREVIEW=1 sh ./scripts/build_docs.sh apm-agent-rum-js ./docs ./build",
     "lint": "run-p --silent lint:*",
+    "lint:commit": "scripts/lint-commits.sh",
```

Comment:
Its better to not do this here instead move CI related things to `ci/scripts/lint.sh`, This script is not so useful for dev workflow.